src/TerrainRenderer.cpp

```diff
@@ -763,16 +736,21 @@ void TerrainRenderer::Draw(const GameWorldView& gwv, unsigned int* water)
 
     lastOffset = PointI(0, 0);
 
+    // Arrays aktivieren
+    glEnableClientState(GL_VERTEX_ARRAY);
+    glEnableClientState(GL_TEXTURE_COORD_ARRAY);
+    glEnableClientState(GL_COLOR_ARRAY);

```

Comment:
ah youre right, I only have seen that you removed some of those disable lines ;-)